src/drive/lib/encryption.js

```diff
@@ -1,3 +1,80 @@
+const stringToArrayBuffer = string => {
+  var encoder = new TextEncoder('utf-8')
+  return encoder.encode(string)
+}
+
+const asDerivableKey = async (
+  baseMaterial,
+  keyDerivationAlgorithm = 'PBKDF2'
+) => {
+  let usableMaterial
+  if (baseMaterial instanceof CryptoKey) {
+    usableMaterial = await window.crypto.subtle.exportKey('raw', baseMaterial)
```

Comment:
Shouldn't it be `jwt` instead of `raw`?

Reply:
Not this time. This is a way of extracting the raw key of a CryptoKey in order to use it again in another derivation. Nonetheless, `jwk` might also work but the JSON structure is not needed for this internal operation